blog/2024-03-19-TensorRT-LLM.md

```diff
@@ -46,61 +45,65 @@ We picked 3 hardware platforms to run the test on, based on Jan's userbase's sel
 
 We didn't bother including low-spec machines: TensorRT-LLM is meant for performance, and simply doesn't work on lower grade Nvidia GPUs, or computers without GPUs.
 
-TensorRT-LLM provides blazing fast performance at the cost of [memory usage](https://nvidia.github.io/TensorRT-LLM/memory.html). This means that the performance improvements only show up in higher-range GPUs with larger VRAMs. 
+TensorRT-LLM provides blazing fast performance at the cost of [memory usage](https://nvidia.github.io/TensorRT-LLM/memory.html). This means that the performance improvements only show up in higher-range GPUs with larger VRAMs.
 
-We've found that [llama.cpp](https://github.com/ggerganov/llama.cpp) does an incredible job of democratizing inference to the [GPU Poor](https://www.semianalysis.com/p/google-gemini-eats-the-world-gemini#the-gpu-poor) with CPU-only or lower-range GPUs. Huge shout outs to the [llama.cpp maintainers](https://github.com/ggerganov/llama.cpp/graphs/contributors) and the [ggml.ai](https://ggml.ai/) team.   
+We've found that [llama.cpp](https://github.com/ggerganov/llama.cpp) does an incredible job of democratizing inference to the [GPU Poor](https://www.semianalysis.com/p/google-gemini-eats-the-world-gemini#the-gpu-poor) with CPU-only or lower-range GPUs. Huge shout outs to the [llama.cpp maintainers](https://github.com/ggerganov/llama.cpp/graphs/contributors) and the [ggml.ai](https://ggml.ai/) team.
 
 :::
 
-We chose the popular Mistral 7b model to run on both GGUF and TensorRT-LLM, picking comparable quantizations. 
+We chose the popular Mistral 7b model to run on both GGUF and TensorRT-LLM, picking comparable quantizations at `INT4`.
 
 #### llama.cpp Setup
-- For llama.cpp, we used `Mistral-7b-q4_k_m`
-- [ ] Fill in `ngl` params, GPU offload etc
+
+- llama.cpp commit [15499eb](https://github.com/ggerganov/llama.cpp/commit/15499eb94227401bdc8875da6eb85c15d37068f7)
+- We used `Mistral-7b-q4_k_m` in `GGUF` with `ngl` at `100`
 
 #### TensorRT-LLM Setup
+
+- TensorRT-LLM version [0.7.1](https://github.com/NVIDIA/TensorRT-LLM/releases/tag/v0.7.1) and build on Windows
 - For TensorRT-LLM, we used `Mistral-7b-int4 AWQ`
 - We ran TensorRT-LLM with `free_gpu_memory_fraction` to test it with the lowest VRAM consumption (performance may be affected)
-- Note: We picked AWQ for TensorRT-LLM as a handicap as AWQ supposedly sacrifices performance for quality
+- Note: We picked AWQ for TensorRT-LLM as a handicap as AWQ supposedly sacrifices quality for performance
```

Comment:
@hiro-v Can I get this right: 

- AWQ sacrifices quality for performance
- If we do so, then aren't we handicapping GGML instead? (in terms of speed)